test_scripts/Defects/8_1/3858/common_3858.lua

```diff
@@ -0,0 +1,156 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+
+--[[ Module ]]
+local m = {}
+
+--[[ Proxy Functions ]]
+m.start = actions.start
+m.preconditions = actions.preconditions
+m.postconditions = actions.postconditions
+m.getMobileSession = actions.getMobileSession
+m.getHMIConnection = actions.getHMIConnection
+m.registerApp = actions.registerApp
+m.activateApp = actions.activateApp
+m.getHMIAppId = actions.app.getHMIId
+
+--[[ Common Variables ]]
+m.tcs = {
```

Comment:
I'm able to understand the other abbreviations used here(`CICS`, `PI`) but what does `tcs` stand for?

Reply:
No, that makes sense. Thanks for clarifying